go/teams/loader.go

```diff
@@ -679,6 +690,12 @@ func (l *TeamLoader) load2CheckReturn(ctx context.Context, arg load2ArgT, res *k
 			return err
 		}
 	}
+	if arg.needKeyGeneration > 0 {
```

Comment:
Should this be `arg.needKBFSKeyGeneration.Generation`?

Reply:
Woops!